NetSerializer/NetSerializer.csproj

```diff
@@ -1,68 +1,25 @@
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>NetSerializer</RootNamespace>
-    <AssemblyName>NetSerializer</AssemblyName>
-    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
-  </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
-    <DebugSymbols>true</DebugSymbols>
-    <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
-    <OutputPath>bin\Debug\</OutputPath>
-    <DefineConstants>TRACE;DEBUG;GENERATE_DEBUGGING_ASSEMBLY</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
+    <TargetFrameworks>netstandard2.1;net45</TargetFrameworks>
+    <Description>A very fast and minimal serializer</Description>
+    <Company>Tomi Valkeinen</Company>
+    <Copyright>Copyright © 2012-2016 Tomi Valkeinen</Copyright>
+    <Version>4.1.0</Version>
+    <PackageLicenseExpression>MPL-2.0</PackageLicenseExpression>
+    <Authors>$(Company)</Authors>
+    <PackageTags>Serializer</PackageTags>
+    <RepositoryUrl>https://github.com/tomba/netserializer</RepositoryUrl>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Prefer32Bit>false</Prefer32Bit>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
-    <DebugType>pdbonly</DebugType>
-    <Optimize>true</Optimize>
-    <OutputPath>bin\Release\</OutputPath>
-    <DefineConstants>TRACE</DefineConstants>
-    <ErrorReport>prompt</ErrorReport>
-    <WarningLevel>4</WarningLevel>
-    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <Prefer32Bit>false</Prefer32Bit>
+
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' ">
+    <DefineConstants Condition="'$(TargetFramework)' == 'net45'">GENERATE_DEBUGGING_ASSEMBLY</DefineConstants>
   </PropertyGroup>
+
   <ItemGroup>
-    <Reference Include="System" />
-  </ItemGroup>
-  <ItemGroup>
-    <Compile Include="Settings.cs" />
-    <Compile Include="TypeIDList.cs" />
-    <Compile Include="TypeSerializers\ArraySerializer.cs" />
-    <Compile Include="TypeData.cs" />
-    <Compile Include="TypeSerializers\NullableSerializer.cs" />
-    <Compile Include="TypeSerializers\DictionarySerializer.cs" />
-    <Compile Include="TypeSerializers\GenericSerializer.cs" />
-    <Compile Include="TypeSerializers\EnumSerializer.cs" />
-    <Compile Include="TypeDictionary.cs" />
-    <Compile Include="Helpers.cs" />
-    <Compile Include="ITypeSerializer.cs" />
-    <Compile Include="Serializer.cs" />
-    <Compile Include="Primitives.cs" />
-    <Compile Include="TypeSerializers\NoOpSerializer.cs" />
-    <Compile Include="TypeSerializers\ObjectSerializer.cs" />
-    <Compile Include="TypeSerializers\PrimitivesSerializer.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
+    <PackageReference Include="Microsoft.NETFramework.ReferenceAssemblies" Version="1.0.0-preview.2">
```

Comment:
Hmm, what is this reference? A ref to "preview.2" doesn't sound good.

Reply:
It's to allow the .NET Core SDK to build the project targeting .NET Framework on mac/Linux, because they can't use regular Framework (you can also make it rely on Mono being installed but this is more elegant and less hacky).

It's preview because .NET Core 3 is also still preview.